strawberryfields/gbs/plot.py

```diff
@@ -0,0 +1,254 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+Plotting and visualization
+==========================
+
+**Module name:** :mod:`strawberryfields.gbs.plot`
+
+.. currentmodule:: strawberryfields.gbs.plot
+
+This module provides functionality for visualizing graphs, subgraphs, and point processes. Graphs
+are plotted using the Kamada-Kawai layout with an aspect ratio of 1:1. The module uses a custom
+Strawberry Fields colour scheme. The standard scheme for graphs uses green nodes and grey edges,
+whereas the scheme for subgraphs uses red nodes and edges.
+
+.. autosummary::
+    plot
+    plot_subgraph
+
+Code details
+^^^^^^^^^^^^
+"""
+from typing import Optional, Tuple
+
+import networkx as nx
+
+
+def _node_coords(graph: nx.Graph, l: dict) -> Tuple:
+    """Converts coordinates for the graph edges for plotting purposes.
+
+    Args:
+        graph (nx.Graph): input graph
+        l (dict[int, float]): dictionary of nodes and their respective coordinates, can be
+        generated using a NetworkX `layout <https://networkx.github.io/documentation/latest/
+        reference/drawing.html#module-networkx.drawing.layout>`__
+
+    Returns:
+         Dict[str, list]: lists of x and y coordinates accessed as keys of a dictionary
```

Comment:
```suggestion
         dict[str, list]: lists of x and y coordinates accessed as keys of a dictionary
```
Never know whether to capitalize or not